File: Periodic Table/Halogen.py
'''
Created on Dec 15, 2013

@author: thomas
'''
from NonMetal import NonMetal

class Halogen(NonMetal):

    CHARGE = 3

    def __init__(self, atomicNumber, atomicMass):
        self.atomicNumber = atomicNumber
        self.atomicMass = atomicMass

fluorine = Halogen(9, 18.998403)
chlorine = Halogen(17, 35.453)
bromine = Halogen(35, 79.904)
iodine = Halogen(53, 126.905)
astatine = Halogen(85, 210)

File: Java/src/Table/Halogen.java
package Table;

/**
 * Created by Thomas on 1/28/14.
 */
public class Halogen extends NonMetal {

    public int atomicNumber;
    public double atomicMass;

    public Halogen(int aN, double aM){
        int atomicNumber = aN;
        double atomicMass = aM;
    }

    Halogen fluorine = new Halogen(9, 18.998403);
    Halogen chlorine = new Halogen(17, 35.453);
    Halogen bromine = new Halogen(35,79.904);
    Halogen iodine = new Halogen(53, 126.905);
    Halogen astatine = new Halogen(85, 210);
}